src/ui/components/LoadingText/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "~amo/css/inc/vars";
+
+@keyframes placeHolderShimmer {
+  0% {
+    opacity: 0.5;
+  }
+
+  25% {
+    opacity: 0.15;
+  }
+
+  50% {
+    opacity: 0.75;
+  }
+
+  75% {
+    opacity: 0.15;
+  }
+
+  100% {
+    opacity: 0.5;
+  }
+}
+
+.LoadingText {
+  animation: placeHolderShimmer 3s infinite cubic-bezier(0.65, 0.05, 0.36, 1);
+  background: $text-color-default;
+  display: inline-block;
+  height: 1rem;
+  line-height: 1;
+  margin: 0;
+  vertical-align: middle;
+}
+
+@for $percentage from 1 through 100 {
```

Comment:
If this is controlled only by client-side code then setting style props directly is the answer for this sort of thing, since react should poke the style properties directly and does not cause CSP issues. See also https://github.com/mozilla/addons-frontend/issues/2321

The only caveat to using style props is for server rendering because the initial server render would use inline-styles and they *would* be blocked by CSP.